app/routes/articles/components/ArticleEditor.tsx

```diff
@@ -64,10 +64,10 @@ const ArticleEditor = ({
         }
       />
       <NavigationTab
-        title="Rediger artikkel"
+        title={isNew ? 'Ny artikkel' : 'Redigerer: ' + article.title}
```

Comment:
Looking at the code above:
```ts
      <Helmet
        title={
          !isNew && article ? `Redigerer: ${article.title}` : 'Ny artikkel'
        }
      />
```
It seems that you might get errors here if `article` is undefined while somehow editing it? 
```suggestion
        title={isNew ? 'Ny artikkel' : 'Redigerer: ' + article?.title}
```

Reply:
Good catch!